static/styles/explorer.scss

```diff
@@ -1211,26 +1211,51 @@ html[data-theme='pink'] {
             flex-grow: 1;
             flex-shrink: 1;
             img {
+                width: 1000px;
+                aspect-ratio: 1000 / 800;
```

Comment:
nice! does this stop it temporarily popping a small window and then resizing once the images load? Been noticing that recently during presentations...

Reply:
Yes!

Reply:
(At least, that's the hope)